src/Common/src/Interop/Windows/mincore/Interop.DuplicateHandle.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.InteropServices;
+using System.Security;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        [DllImport(Libraries.Handle, SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
```

Comment:
MarshalAs(UnmanagedType.Bool) is the default for bool (and matches CloseHandle returning a BOOL), so this shouldn't be necessary.